packages/frontend-2/lib/settings/graphql/queries.ts

```diff
@@ -12,3 +12,21 @@ export const settingsSidebarWorkspacesQuery = graphql(`
     }
   }
 `)
+
+export const settingsWorkspacesMembersQuery = graphql(`
+  query SettingsWorkspacesMembers($workspaceId: String!) {
+    workspace(id: $workspaceId) {
+      team {
```

Comment:
https://www.notion.so/speckle/General-Web-Code-Review-Guide-1f2bf3dfea634460b3778ac3cf4aa1bb?pvs=4#d40e40efc32f4e5ea67b68e848aadcd0

Reply:
Thanks :) I plan to make a PR to make a few adjustments, I've also noticed some other small things like fragment names not really matching anymore, Ill include fixing this